plugins/plugin_manager/plugin_view.lua

```diff
@@ -166,17 +161,24 @@ function PluginView:draw()
     return self:draw_loading_screen(self.progress and self.progress.label, self.progress and self.progress.percent)
   end
 
+  renderer.draw_rect(self.position.x, self.position.y, self.size.x, 1, style.dim)
 
   local ox, oy = self:get_content_offset()
   oy = oy + lh * self.offset_y
 
   local x, y = ox + style.padding.x, oy
   for i, v in ipairs(self.plugin_table_columns) do
     common.draw_text(style.font, style.accent, v, "left", x, self.position.y, self.widths[i], lh)
-    x = x + self.widths[i] + style.padding.x
+    renderer.draw_rect(x + self.widths[i], self.position.y, 1, self.size.y, style.dim)
+    x = x + self.widths[i] + style.padding.x + 2
   end
 
+  renderer.draw_rect(self.position.x, self.position.y + lh * self.offset_y, self.size.x, 1, style.dim)
+  
+  local x = self.position.x + self.size.x - style.padding.x - style.font:get_width("Search")
+  common.draw_text(style.font, style.dim, "Search: Ctrl+F", "right", x, self.position.y, style.font:get_width("Ctrl+F"), lh)
```

Comment:
This, I like. But you should probably use `keymap.get_binding` to display this. Also, why `- style.font:get_width("Search")`? Shouldn't you just pile it all into the second line, and make the width passed to `draw_text` the width of the whole string?